src/EnergyPlus/WeatherManager.cc

```diff
@@ -2406,13 +2406,13 @@ namespace Weather {
                         ErrorsFound = true;
                     }
 
-                    if (DirectRad < 9999.0 && DirectRad < 0.0) {
+                    if (DirectRad < 0.0) {
                         ShowSevereError(state, format("{}: {}", routineName, state.dataEnvrn->WeatherFileLocationTitle));
                         ShowContinueError(state, format("Direct Radiation ({:.2R}) is out of range [0.0, -]", DirectRad));
                         ErrorsFound = true;
                     }
 
-                    if (DiffuseRad < 9999.0 && DiffuseRad < 0.0) {
+                    if (DiffuseRad < 0.0) {
```

Comment:
[src/EnergyPlus/WeatherManager.cc:2409]:(style),[redundantCondition],Redundant condition: The condition 'DirectRad < 9999.0' is redundant since 'DirectRad < 0.0' is sufficient.
[src/EnergyPlus/WeatherManager.cc:2415]:(style),[redundantCondition],Redundant condition: The condition 'DiffuseRad < 9999.0' is redundant since 'DiffuseRad < 0.0' is sufficient.